src/components/blockHelpers.tsx

```diff
@@ -37,6 +37,11 @@ export const CONTENT_MAX_WIDTH = {
   },
 }
 
+// New content max width
```

Comment:
do we need this comment? 😬

Reply:
Hehe nope, not necessary :) Just a reminder for us to remove the old max width later on.